app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,8 @@
+class FoodsController < ApplicationController
+  def index
+    if params[:q]
+      @food = params[:q]
```

Comment:
The query used in the API call should be available as part of that response (`json[:foodSearchCriteria][:query]`); I don't think you need this instance variable.